tests/EmblemVaultSdk.test.ts

```diff
@@ -18,7 +18,7 @@ describe('EmblemVaultSDK', () => {
         test('should get curated contracts', async () => {
             const sdk = new EmblemVaultSDK(apiKey);
             const contracts = await sdk.fetchCuratedContracts();
-            expect(contracts).toBeInstanceOf(Array);
+            expect(Array.isArray(contracts)).toBeTruthy();
```

Comment:
Once I switched to using Node's fetch, the object returned by `response.json`, even though an `Array`, was failing Jest's `.toBeInstanceOf(Array)` test. It seems the reason has to do with prototype hierarchy created in another frame. I don't fully understand it, but here's some info:

https://stackoverflow.com/questions/22289727/difference-between-using-array-isarray-and-instanceof-array

https://stackoverflow.com/questions/76007003/jest-tobeinstanceof-expected-constructor-array-received-constructor-array




See this Stackoverflow thread